data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/FindExistingNotInvalidTransactionIds.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "record",
+  "name": "FindExistingNotInvalidTransactionIds",
+  "doc": "A request to fetch transaction that are not invalid (i.e. Verified or Unverified).",
+  "namespace": "net.corda.data.ledger.persistence",
```

Comment:
Looking forward, do we think there would be any benefit of having the ability to find transaction in a set of statuses? @relyafi @blsemo.

If not, then we should go with what Nandor has done, otherwise we can make it more conditional.

Reply:
@lankydan I'd imagine for the most part, people would be interested in valid transactions only? But at the state level, a lot of our queries are implicitly filtered on that already by virtue of only being in the outputs tables if they are verified.

Reply:
That implicit `NotInvalid` is a bit strange. Accepting a set of statuses would make it more generic.
Also, do we need `Existing` in the name?

Reply:
Okay, I'm happy to modify it to be more generic.